### DISCUSSION TITLE
Shall we warn about DOM append ops in for loop?

### DISCUSSION THREAD

**Discussion by jerryc05** [eslint/eslint#16245] [Ideas]
As the title suggests, since appending DOM is expensive.

solutions can be collecting all elements to append into a list and then append all of them after the for loop in one function call (e.g. `append()`), or using a `DocumentFragment`

**Comment by nzakas** [comment]
If you are suggesting that ESLint core add a rule to catch this, the answer is no. We don't include runtime-specific rules in the core.

You can, however, create your own rule to do this.
